docs/releases/release-1.0.md

```diff
@@ -0,0 +1,50 @@
+# Kubeflow Pipelines 1.0 Release Notes
+
+## Versioning policy
+
+Kubeflow Pipelines 1.0 is the first major release of Kubeflow Pipelines. It follows the Kubeflow Pipelines [versioning policy](https://github.com/kubeflow/pipelines/blob/master/docs/releases/versioning-policy.md).
+
+## Deployment options
+
+There are two available ways to [deployment Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+
+* [Standalone Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/installation/standalone-deployment/), which allows users the maximum flexibility in configuring the Kubernetes clusters and a Kubeflow Pipelines instance.
+
+* [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs), which minimizes the user effort in deploying a Kubeflow pipelines instance.
+
+## Core features and APIs
+
+Core features and APIs are the fundamental functionalities of Kubeflow Pipelines. The most fundamental functionality is to manage the Kubeflow Pipelines resources (that is, pipeline, pipeline version, run, job, experiment) via the operations listed below. Active supports are available for core features and APIs.
+
+|                  | Upload | Create | Get | List | Delete | Archive | Unarchive | Enable | Disable | Terminate | Retry |
+|:----------------:|:------:|:------:|:---:|:----:|:------:|:-------:|:---------:|:------:|:-------:|:---------:|:-----:|
+| Pipeline         | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Pipeline Version | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+| Run              | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |        |         | - [x]     | - [x] |
+| Job              | - [x]  |  - [x] |- [x]|- [x] | - [x]  |         |           | - [x]  | - [x]   |
+| Experiment       | - [x]  |  - [x] |- [x]|- [x] | - [x]  | - [x]   | - [x]     |
+
+* Refer to the Kubeflow Pipelines [introduction]((https://www.kubeflow.org/docs/pipelines/overview/)) for
+  - An [overview](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) and [main concepts](https://www.kubeflow.org/docs/pipelines/overview/concepts/), for example, what is a pipeline or run in Kubeflow Pipelines.
+  - How to manage the Kubeflow Pipelines resources via the the [Kubeflow Pipelines UI](https://www.kubeflow.org/docs/pipelines/overview/interfaces/).
+* Refer to the Kubeflow Pipelines [SDK samples and tutorials](https://www.kubeflow.org/docs/pipelines/tutorials/sdk-examples/)
```

Comment:
> SDK client (helper methods)
I think that some of them can be deprecated in favor of the generated APIs (kfp.Client().runs, kfp.Client().pipelines, etc).
I only ever use create_run_from_pipeline_func helper.
-- Ark-kun

How do we inform users these helper methods are not core?